src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationParameters.cs

```diff
@@ -154,7 +159,8 @@ private async Task<AuthenticationParameters> CreateFromResourceUrlCommonAsync(Ur
 
             try
             {
-                await _httpManager.SendGetAsync(
+                HttpManager httpManager = new HttpManager();
```

Comment:
will this mean that we're getting a different HttpClient than the default one?  I need to dig into the httpmanager constructor to validate.  I"m mainly concerned if someone has decorated their httpclientfactory with proxy information then it might not get through here.

Reply:
This is fine, because this is a completely separate helper class, has nothing to do with ADAL. You're not even creating an AuthenticationContext, so a custom HttpClient can't even be used. 

If we weren't already in on a stable version, I would propose to remove this API completely.

Reply:
To be honest, an HttpManager isn't even needed here (see how many nulls are used in initialization) -  I could create a simple HttpClient to make a GET call. The only minor advantage of HttpManager is a bit of extra logging.